### PR TITLE
ADEN-5959 | Remove AmazonMatch bidder

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/ADEN-5959

### CODE REVIEW

File: extensions/wikia/AdEngine/js/spec/AdEngineRunner.spec.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ describe('ext.wikia.adEngine.adEngineRunner', function () {
 	function noop() {
 	}
 
+	// TODO tests
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/spec/AdEngineRunner.spec.js]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[project-id]: wikia:app

File: extensions/wikia/AdEngine/js/spec/lookup/services.spec.js [hunk 2]
```diff
@@ -2,18 +2,19 @@
 describe('Method ext.wikia.adEngine.lookup.services', function () {
 	'use strict';
 
+	// TODO tests
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/spec/lookup/services.spec.js]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[project-id]: wikia:app

**Comment by WikiaTech** [comment]
SonarQube analysis reported 46 issues
* ![MAJOR][MAJOR] 36 major
* ![MINOR][MINOR] 2 minor
* ![INFO][INFO] 8 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/92a19e0878a9198959467d7f07b3811f191438fe/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/92a19e0878a9198959467d7f07b3811f191438fe/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/92a19e0878a9198959467d7f07b3811f191438fe/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/92a19e0878a9198959467d7f07b3811f191438fe/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/92a19e0878a9198959467d7f07b3811f191438fe/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L122](https://github.com/Wikia/app/blob/92a19e0878a9198959467d7f07b3811f191438fe/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L122): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L123](https://github.com/Wikia/app/blob/92a19e0878a9198959467d7f07b3811f191438fe/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L123): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L197](https://github.com/Wikia/app/blob/92a19e0878a9198959467d7f07b3811f191438fe/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L197): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L221](https://github.com/Wikia/app/blob/92a19e0878a9198959467d7f07b3811f191438fe/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L221): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L221](https://github.com/Wikia/app/blob/92a19e0878a9198959467d7f07b3811f191438fe/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L221): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 44 issues
* ![MAJOR][MAJOR] 36 major
* ![MINOR][MINOR] 2 minor
* ![INFO][INFO] 6 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/d551afb633f89d11c038a55b5c2cf1a6978db81d/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/d551afb633f89d11c038a55b5c2cf1a6978db81d/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/d551afb633f89d11c038a55b5c2cf1a6978db81d/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/d551afb633f89d11c038a55b5c2cf1a6978db81d/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/d551afb633f89d11c038a55b5c2cf1a6978db81d/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L122](https://github.com/Wikia/app/blob/d551afb633f89d11c038a55b5c2cf1a6978db81d/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L122): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L123](https://github.com/Wikia/app/blob/d551afb633f89d11c038a55b5c2cf1a6978db81d/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L123): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L197](https://github.com/Wikia/app/blob/d551afb633f89d11c038a55b5c2cf1a6978db81d/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L197): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L221](https://github.com/Wikia/app/blob/d551afb633f89d11c038a55b5c2cf1a6978db81d/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L221): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L221](https://github.com/Wikia/app/blob/d551afb633f89d11c038a55b5c2cf1a6978db81d/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L221): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**